neqo-http3/src/server.rs

```diff
@@ -876,8 +945,7 @@ mod tests {
                         .unwrap();
                     data_received += 1;
                 }
-                Http3ServerEvent::StateChange { .. } => {}
-                Http3ServerEvent::PriorityUpdate { .. } => {}
+                _ => {}
```

Comment:
This is why you want to mark off webtransport-specific events.  We should not be using _ in matches like this (it makes things more brittle).